packages/evm/contracts/incentives/ERC20VariableIncentive.sol

```diff
@@ -51,6 +51,7 @@ contract ERC20VariableIncentive is AERC20VariableIncentive, RBAC {
         reward = reward_;
         limit = limit_;
         totalClaimed = 0;
+        claims += 1;
```

Comment:
I think this `claims += 1` is in the wrong place?

Should happen on `claim()`, not `initialize()`